codeownerizer.go

```diff
@@ -102,3 +105,39 @@ func AddUngrantedOwners(ctx context.Context, api *github.Client, org string, rep
 
 	return nil
 }
+
+func ListTeams(ctx context.Context, api *github.Client, org string, repo string) ([]*github.Team, error) {
+	allTeams := []*github.Team{}
+	opts := &github.ListOptions{PerPage: 100}
+	for {
+		teams, resp, err := api.Repositories.ListTeams(ctx, org, repo, opts)
+		if err != nil {
+			return nil, err
+		}
```

Comment:
Note, `err` here can be `429 Too Many Requests` 🙏 Maybe we can slip in a sleep in the future